fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -468,6 +468,22 @@ public static ConstantOperator fromUnixTime(ConstantOperator unixTime) throws An
         return ConstantOperator.createVarchar(dl.toString());
     }
 
+    @ConstantFunction.List(list = {
+            @ConstantFunction(name = "from_unixtime_ms", argTypes = {BIGINT}, returnType = VARCHAR)
```

Comment:
do we need consider int arg type?

Reply:
No, because from_unixtime_ms is a millisecond timestamp, which must be >10 digits, so the Int type is not needed.